internal/controller/replication.storage/volumereplication_controller.go

```diff
@@ -173,7 +173,7 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		vgr, vgrc, vgrErr = r.getVolumeGroupReplicationDataSource(logger, nameSpacedName)
 		if vgrErr != nil {
 			if errors.IsNotFound(vgrErr) && !instance.DeletionTimestamp.IsZero() {
-				logger.Info("volumeGroupReplication resource not found, as volumeReplication resource is getting garbage collected")
+				logger.Info("volumeGroupReplicationContent resource not found, as volumeReplication resource is getting garbage collected")
```

Comment:
vgrErr may denote both volumeGroupReplication and volumeGroupReplicationContent?

Reply:
It can, though the possibility of that happening is _almost_ nil and can only happen if someone manually delete the VGR by removing the finalizers from it.
But, yeah no harm in having both the CRs mentioned in the log.

Reply:
Hence, only mentioned vgrContent in the log.